helper/clientInitialize.go

```diff
@@ -104,7 +104,9 @@ func getClient() *appknox.Client {
     return client
 }
 
-
+func GetClient() *appknox.Client {
+    return getClient()
+}
```

Comment:
Remove this function. Use `getClient` will be accessible from any helper function.